rich/_log_render.py

```diff
@@ -51,7 +51,12 @@ def __call__(
         row: List["RenderableType"] = []
         if self.show_time:
             log_time = log_time or console.get_datetime()
-            log_time_display = log_time.strftime(time_format or self.time_format)
+            if time_format:
```

Comment:
The `time_format` parameter should probably also take a callable, for sake of consistency. Then the logic can be simplified a little.

```python
time_format = time_format or self.time_format
if isinstance(time_format, str):
    log_time_display = Text(log_time.strftime(time_format))
else:
    log_time_display = time_format(log_time)
```

Reply:
I can change the typing of the time_format argument to the __call__ function to allow a callable but since time_format is coming directly from the logging packages Formatter class and is typed as `datefmt: Optional[str]` then we'll never actually get a callable.

I agree the conditional is cleaner and wasn't a big fan of my three way conditional so do you want to proceed with changing the typing and then not worry that the theoretically callable type will never be used?